src/domain/core/src/entities/metadata_chain.rs

```diff
@@ -487,16 +493,14 @@ pub enum AppendValidationError {
 ///////////////////////////////////////////////////////////////////////////////
 
 #[derive(Error, PartialEq, Eq, Debug)]
-#[error("Invalid event: {message}: {event:?}")]
+#[error("Invalid event: {message}")]
 pub struct InvalidEventError {
-    event: MetadataEvent,
```

Comment:
Why are we removing this? This field will gets logged by our error handlers and is a source of valuable information. If anything it probably should be made `pub` to underline that its meat to be used by those handling this error.

Reply:
After examining the code, came to the conclusion that this field is used solely for logging -- so I included a representation of the event in the message:

https://github.com/kamu-data/kamu-cli/pull/473/commits/ef9517e5da0bd248d83f61017824e1ae357b8ae2#diff-82e550963850c5eab67f442e4a683a96f7f6a7eb462d8e04d5ec0788679a24ffR19-R24

Reply:
Yes, please Box the event, there are useful fields to see in platform server logs

Reply:
I see, the message now includes a full debug trace of the event already, so this looks fine then.

Reply:
Added code comments to make this decision clearer for future readers

Reply:
Why do we have two different approaches between `InvalidEventError` and `NoOpEventError`?

Personally I like having a boxed `event` in the error rather than pre-formatting it into the message, as it gives more flexibility on how to present the error.

Reply:
> Personally I like having a boxed event in the error rather than pre-formatting it into the message, as it gives more flexibility on how to present the error.

agree, consistency is paramount -- will use `Box<T>`